packages/serverless-offline-s3/src/index.js

```diff
@@ -17,15 +15,17 @@ const defaultOptions = {
 const omitUndefined = omitBy(isUndefined);
 
 class ServerlessOfflineS3 {
-  constructor(serverless, cliOptions) {
+  constructor(serverless, cliOptions, {log}) {
     this.cliOptions = null;
     this.options = null;
     this.s3 = null;
     this.lambda = null;
     this.serverless = null;
+    this.log = null;
 
     this.cliOptions = cliOptions;
     this.serverless = serverless;
+    this.log = log;
```

Comment:
same as the other one : 


```suggestion
this.log = log || {
  debug: console.debug.bind(console),
  notice: console.log.bind(console),
  warning: console.warn.bind(console)
};```